scripts/source/bdr.js

```diff
@@ -19,8 +19,10 @@ const { exec } = require("child_process");
 const isAbsoluteUrl = require("is-absolute-url");
 
 const fileToMetadata = {};
-const basePath = path.resolve("temp_bdr/docs/docs2/");
-const imgPath = path.resolve("temp_bdr/docs/img/");
+const args = process.argv.slice(2);
+const basePath = path.resolve(args[0], "docs/");
+const imgPath = path.resolve(args[0], "docs/img/");
```

Comment:
minor nit, but I'd probably build this on basePath vs referencing args[0] twice; just makes it a wee bit easier to read backwards